build.gradle.kts

```diff
@@ -270,6 +270,25 @@ publishing {
             artifact(javadocJar)
             artifact(sourcesJar)
 
+            pom {
+                name.set("JDA")
+                issueManagement {
+                    url.set("https://github.com/DV8FromTheWorld/JDA/issues")
+                    system.set("Github")
+                }
+                scm {
+                    url.set("https://github.com/DV8FromTheWorld/JDA")
+                    connection.set("scm:git:git://github.com/DV8FromTheWorld/JDA")
+                    developerConnection.set("scm:git:ssh://github.com/DV8FromTheWorld/JDA.git")
```

Comment:
Maybe it's better to not use ssh for it, not everyone has git over ssh configured

Reply:
Both ssh and https git connections are listed.